nim/basolato/config.yaml

```diff
@@ -1,15 +1,11 @@
 framework:
   github: itsumura-h/nim-basolato
-  version: 0.5
+  version: 0.6
 
 files:
   - /usr/src/app/server
 
 deps:
-  - openssh-client
-  - openssl
-  - pcre
-  - pcre-dev
   - bsd-compat-headers
-
-build_opts: '--define: usePcreHeader --passL: "/usr/lib/libpcre.a"'
+  
+build_opts: '--threadAnalysis:off'
```

Comment:
does other `nim` ** framework** not need this config ?

Reply:
@waghanza 
It causes an error especially in Basolato .
I don't know whether other nim framework not need this.
I think we need to consider which web frameworks were added at the time this description was added. Perhaps that framework need this config.

Reply:
@itsumura-h on `alpine` this flag is **required**